static/js/country_cards.js

```diff
@@ -183,6 +190,65 @@ const setSearch = (event) => {
     selectBtn.firstElementChild.innerText = 'Filter by Region';
 }
 
+const changeGridLayout = (event) => {
+    // Make sure target is always the button
+    const btn = event.target.closest('button');
+
+    switch (btn.classList[1]) {
+        case 'tile-btn-1':
+            countryCardContainer.style.gridTemplateColumns = 'repeat(1, minmax(280px, 1fr))';
+            break;
+        case 'tile-btn-2':
+            countryCardContainer.style.gridTemplateColumns = 'repeat(2, minmax(280px, 1fr))';
+            break;
+        case 'tile-btn-3':
+            countryCardContainer.style.gridTemplateColumns = 'repeat(3, minmax(280px, 1fr))';
+            break;
+        case 'tile-btn-4':
+            countryCardContainer.style.gridTemplateColumns = 'repeat(4, minmax(280px, 1fr))';
+            break;
+    }
+}
+
+const resetGridLayout = (event) => {
```

Comment:
We kunnen het hier later over hebben, maar wat betreft dit stukje, Het werkt dus prima, alleen ik zou wel goed nadenken over je aanpak.

In principe weet je wat er gedaan moet worden, en dat voer je ook uit. Maar je moet een stap dieper denken over de functionaliteit. In the sense, je kan het voor jezelf versimpelen.

Dus denk aan, Ik heb 4 buttons, twee daarvan toggelen minder tiles, en 2 meer. Enig verschil is dat je op mobiel minder tiles, dan op desktop wil zien.